srcpkgs/q5go/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'q5go'
+pkgname=q5go
+version=2.1.2
+revision=1
+archs="i686 x86_64"
```

Comment:
Nope, figure out cross build. dropping `archs=` is first step.

Reply:
It's getting better :) You can check locally with `./xbps-src -a aarch64 pkg q5go`

Reply:
The package itself shows no documentation on which archs etc... what's the best next step as to figuring out how to get this to run on other archs? That local check is useful too.

Reply:
Usually one adds necessary deps/flags/whatever until it compiles. At that point we assume it runs as well if build system is common. qmake is.
If you hit something you can't figure out, push that and ask.

```
$ xlocate bin/uic
qt5-host-tools-5.15.3+20211001_2        /usr/lib/qt5/bin/uic
```

Reply:
Testing locally using aarch64. Change working so far is I copied the depends to hostmakedepends and now it's compiling. Though I will likely include that qt5-host-tools as a hostmakedepends as well.